src/gpu/nvidia/cudnn_gemm_inner_product_impl.hpp

```diff
@@ -271,6 +296,10 @@ struct cudnn_gemm_inner_product_fwd_impl_t
 
         return status::success;
     }
+
+    ~cudnn_gemm_inner_product_fwd_impl_t() {
+        if (with_f32_sum_) { cudnnDestroyTensorDescriptor(bias_f32_desc_); }
```

Comment:
I can't find where `tensor_descs_` gets destroyed but given there are other additional tensors created but only a new one gets destroyed here is suspicious. My naive understanding is either this destroy call should move to the place where it's already done (I failed to identify it), or other tensor should be destroyed here as well. Could you check it for me, please, which option is better?